src/main/java/mate/academy/spring/dao/impl/UserDaoImpl.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.spring.dao.impl;
+
+import java.util.List;
+import mate.academy.spring.dao.UserDao;
+import mate.academy.spring.exception.DataProcessingException;
+import mate.academy.spring.model.User;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class UserDaoImpl implements UserDao {
+    private final SessionFactory sessionFactory;
+
+    public UserDaoImpl(SessionFactory sessionFactory) {
```

Comment:
please either put annotation over the constructor everywhere - or put it nowhere